### PR TITLE
Chore and details

### PR DESCRIPTION
**Pull request by pylapp**
- add rules in .gitignore
- update README with details about workflow to follow and issues

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,4 @@ functions/lib/
 serviceAccount.json
 serviceAccount.dev.json
 firebase-debug.log
+.firebase/hosting.YnVpbGQ.cache
```
Comment (LostInBrittany) [inline .gitignore]: Maybe we could blacklist all the .firebase/*.cache ?

File: README.md [hunk 2]
```diff
@@ -30,27 +30,37 @@ Our goal is to allow event organizers to set up professional conference website
 
 ## Getting Started
 1. [Fork repository](https://github.com/gdg-x/hoverboard/fork) and clone it locally
-1. Setup Environment
+2. Setup Environment
```
Comment (LostInBrittany) [inline README.md:33]: Numérotation in Markdown doesn't work like that ;)

All the elements in a numbered list share the same number.

I.e.

```
1. Hello
1. world
```
Donne : 
1. Hello
1. world

**Review by LostInBrittany — COMMENTED** [review]
I have posted some suggestions in the PR comments

File: .gitignore (hunk 1 above)
Comment (pylapp) [reply .gitignore]: Good idea!

File: README.md (hunk 2 above)
Comment (pylapp) [reply README.md:33]: You can also use different values for the chips (according to https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet, "Lists" section) :)

**Comment by pylapp** [comment]
- fix hyperlinks point to Brest web site
- update hyperlinks pointing to partnership folders (now public)
- now the CFP page is opened only one time by clicking on the dedicated button

**Pull request merged**